routes/index.js

```diff
@@ -609,6 +598,22 @@ var routes = {
               updateData.description = 'Mozilla Foundation Monthly Donation';
             }
 
+            // capture recurring stripe transactions in salesforce
+            basket.queue({
+              event_type: "donation",
+              last_name: subscription.customer.sources.data[0].name,
+              email: subscription.customer.email,
+              donation_amount: basket.zeroDecimalCurrencyFix(charge.amount, charge.currency),
+              currency: charge.currency,
+              created: charge.created,
+              recurring: true,
+              frequency: "monthly",
+              service: "stripe",
+              transaction_id: charge.id,
+              subscription_id: subscription.id,
+              project: charge.metadata.thunderbird ? "thunderbird" : ( charge.metadata.glassroomnyc ? "glassroomnyc" : "mozillafoundation" )
```

Comment:
@ScottDowne is this going to work still? I can't remember if the metadata is applied to the subscription or the charge..

Reply:
(It was wrong, I updated it to check `updateData`)